src/Navbar/Navbar.tsx

```diff
@@ -0,0 +1,31 @@
+import cn from 'classnames';
+import { NavLink } from 'react-router-dom';
+
+export const Navbar = () => (
+  <nav
+    className="navbar is-light is-fixed-top is-mobile has-shadow"
+    data-cy="Nav"
+  >
+    <div className="container">
+      <div className="navbar-brand">
+        <NavLink
+          to="/"
+          className={({ isActive }) => cn('navbar-item', {
+            'is-active': isActive,
+          })}
+        >
+          Home
+        </NavLink>
+
+        <NavLink
+          to="/tabs"
+          className={({ isActive }) => cn('navbar-item', {
```

Comment:
I think it would be better to create a custom component  for NavLinks to avoid code duplication

Reply:
strongly agree